src/sif/dynamiccost.cc

```diff
@@ -53,9 +53,12 @@ namespace sif {
 // default options/parameters
 namespace {
 
+// max penalty to apply when use tracks
+constexpr float kMaxTrackPenalty = 300.f; // 5 min
```

Comment:
are you going to add a costing option to all the user to set this? a lot of our other penalties work like this like gate penalty and alley penalty

Reply:
I've implemented the same logic as @SvetlanaBayarovich for `living_street`. reasons:
1. for external users it's more convenient to have one clear parameter `use_tracks` to represent preference of track roads;
2. right now `track_penalty` is measured in seconds that we're adding to the transition cost; if we change it in the future, we will have to notify users about that. In case of one `use_tracks` parameter we can tweak `track_factor` and `track_penalty` without affecting users